### DISCUSSION TITLE
Lambda Function URLs

### DISCUSSION THREAD

**Discussion by lopesjeronimo** [Kludex/mangum#255] [General]
It's not a issue, just a testimonial. 

I've just made a very simple "hello world" using FastAPI, Mangum and Serverless with new recently launched AWS Lambda Function URLs.

It worked like a charm.

**Comment by jordaneremieff** [comment]
Always nice to hear about how it's being used. :)

Was not aware of AWS Lambda Function URLs, interesting!
